tsinfer/inference.py

```diff
@@ -640,7 +659,7 @@ def match_samples(
         # we sometimes assume they are in the same order as in the file
 
     manager.match_samples(sample_indexes, sample_times)
-    ts = manager.finalise(simplify=simplify)
+    ts = manager.finalise(do_post_process=post_process, simplify_only=simplify_only)
```

Comment:
I'm not sure there's much point in sending these parameters to the finalise method - could we just do
```
if post_process:
    ts = post_process(...)
return ts
```

Reply:
Oh yes, good to have the extra eyes on this. Although it's a nice idea, it does create a variable name conflict between the `post_process` parameter and the identically named function. Do we want to rename anything to get around this?

Reply:
I can work around it by defining `_post_process` as an internal alias for the `post_process()` function, but it's a bit messy.